File: week-3/Md. Maruf Sarker/Is the Score Consistent.go
// Written by: Md. Maruf Sarker
// Platform: CodeChef
// Problem: https://www.codechef.com/problems/TRUESCORE
// Language: go
// Date: 2023-02-27
// Difficulty Rating: 572

package main

import "fmt"
func main() {
	var t int
	var a, b, c, d int32
	fmt.Scan(&t)

	for i := 0; i < t; i++ {
		fmt.Scan(&a, &b, &c, &d)
		
		if a <= c && b <= d {
			fmt.Println("POSSIBLE")
		}else{
			fmt.Println("IMPOSSIBLE")
		}
	}
}

File: week-3/SABIHA/Is the Score Consistent.c
#include <stdio.h>

int main(void) {
    int t, a, b, c, d ;
    scanf("%d", &t) ;
    for (int i = 1 ; i<=t ; i++){
        scanf("%d %d %d %d", &a, &b, &c, &d) ;
         if (a<=c && d>=b)
        printf("POSSIBLE\n ") ;
    
    else
    printf("IMPOSSIBLE\n ") ;
     }
    return 0;
}